doc/api/child_process.markdown

```diff
@@ -3,8 +3,8 @@
     Stability: 2 - Stable
 
 The `child_process` module provides the ability to spawn child processes in
-a manner that is similar, but not identical, to [`popen(3)`][]. This capability
-is primarily provided by the `child_process.spawn()` function:
+a manner that is similar, but not identical, to popen(3). This capability
```

Comment:
@eljefedelrodeodeljefe Maybe no need in keeping links anymore thus #5073 landed?